datahub-ranger-plugin/src/main/java/com/datahub/authorizer/plugin/ranger/DataHubRangerAuthPlugin.java

```diff
@@ -0,0 +1,59 @@
+package com.datahub.authorizer.plugin.ranger;
+
+import org.apache.ranger.plugin.client.BaseClient;
+import org.apache.log4j.Logger;
+import org.apache.ranger.plugin.service.RangerBaseService;
+import org.apache.ranger.plugin.service.ResourceLookupContext;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Datahub Apache Ranger Plugin.
+ * It assists in creating policies on Apache Ranger Admin Portal.
+ *
+ */
+public class DataHubRangerAuthPlugin extends RangerBaseService {
+    private static Logger log = Logger.getLogger(DataHubRangerAuthPlugin.class.getName());
+
+    /**
+     * This is dummy function. As this plugin doesn't have any configuration
+     * @return A Map with success message
+     * @throws Exception
+     */
+    @Override
+    public Map<String, Object> validateConfig() throws Exception {
+        // We don't have any config to validate
+        return returnSuccessMap();
+    }
+
+    /**
+     * This is dummy function. As this plugin doesn't support the resource lookup
+     * @param resourceLookupContext
+     * @return Empty list of string
+     * @throws Exception
+     */
+    @Override
+    public List<String> lookupResource(ResourceLookupContext resourceLookupContext) throws Exception {
+        // Not supporting resource lookup
+        return Arrays.asList("");
```

Comment:
Should we throw unsupportedoperationexception?

Why is lookup not supported??

Reply:
Done